packages/app-store/routing-forms/pages/form-edit/[...appPages].tsx

```diff
@@ -177,28 +226,55 @@ function Field({
             />
           </div>
           {["select", "multiselect"].includes(hookForm.watch(`${hookFieldNamespace}.type`)) ? (
-            <div className="mt-2 block items-center sm:flex">
-              <div className="w-full">
-                <TextAreaField
-                  disabled={!!router}
```

Comment:
Earlier it was disabled in case the field was coming from Router Form but now it's enabled.
<img width="1176" alt="Screenshot 2023-05-17 at 6 18 31 PM" src="https://github.com/calcom/cal.com/assets/1780212/899b2f63-dcd3-42a8-8541-9dd8cffe7a6f">